addon/components/high-charts.js

```diff
@@ -58,15 +59,16 @@ export default Ember.Component.extend({
   },
 
   draw() {
-    let options = [ get(this, 'buildOptions') ];
-    let mode  = get(this, 'mode');
+    let completeChartOptions = [ get(this, 'buildOptions'), get(this, 'callback') ],
```

Comment:
I definitely appreciate the intent, I'm glad you're concerned about improving readability and collaboration. However, jshint will check for variables being redeclared, so that's a non-issue. 

Plus having multiple declarations makes it harder to maintain – now you have to remember to use a comma. To quote the airbnb style guide:

> Use one `var` declaration per variable. It's easier to add new variable declarations this way, and you never have to worry about swapping out a `;` for a `,` or introducing punctuation-only diffs.

The main thing is consistency. We don't currently do multiple declarations, so I'd like to keep it consistent and not do that.

cc @ahmadsoe